.travis.yml

```diff
@@ -13,8 +13,11 @@
 # - basics - quick things like style and packaging
 # - test - the actual functional tests which are slow
 
+dist: trusty
 language: python
-cache: pip
+cache:
+  - pip
+  - apt: false
```

Comment:
Why exclude apt from the cache? We install several command line tools for testing using the apt packages.